service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingVenueLookUpService.java

```diff
@@ -32,16 +31,14 @@ public HearingVenueLookUpService(ObjectMapper objectMapper) {
     }
 
     private void populateHearingVenueMappings() {
-        if (isEmpty(hearingVenues)) {
-            try {
-                final String jsonContent = ResourceReader.readString("static_data/hearingVenues.json");
-                hearingVenues = objectMapper.reader()
-                    .forType(new TypeReference<List<HearingVenue>>() {})
-                    .readValue(jsonContent);
-
-            } catch (IOException e) {
-                log.error("Unable to parse hearingVenues.json file.", e);
-            }
+        try {
+            final String jsonContent = ResourceReader.readString("static_data/hearingVenues.json");
+            hearingVenues = objectMapper.reader()
+                .forType(new TypeReference<List<HearingVenue>>() {
+                })
+                .readValue(jsonContent);
```

Comment:
Do you know if there is a reason we are reading this differently compared to how we normally do it?
Normally we just do
```java
mapper.readValue(jsonContent, newTypeReference<>(){});
```

Reply:
dont know, but you are right, changed